examples/configuration.nix

```diff
@@ -38,9 +42,8 @@
   # in C.
   services.clightning.enable = true;
   #
-  # Set this to create an onion service by which clightning can accept incoming connections
-  # via Tor.
-  # The onion service is automatically announced to peers.
```

Comment:
Wasn't this correct? I thought we had fixed this in the onionServices PR. `onionService.clightning.enable` defaults to the setting of public and we don't set `onionService.clightning` in `onion-services.nix` nor in `enable-tor.nix`.

Reply:
Sorry, you're totally correct. I've reverted this commit.